### DISCUSSION TITLE
Configurable Operators

### DISCUSSION THREAD

**Discussion by Benjathing** [Biarity/Sieve#161] [Ideas]
We've recently come across an issue when using Sieve over Azure API Management Gateway: there's a rule in there that is preventing a call with "!=" in it from reaching the server. It's being flagged as a potential SQL Injection.

Might we possibly get the ability to configure alternative operators in cases such as these?

**Comment by hasanmanzak** [comment]
Maybe this way?

https://github.com/Biarity/Sieve#creating-your-own-dsl
